components/categ-tool-group.vue

```diff
@@ -33,6 +34,7 @@
                 <b-input-group>
                     <label for="column-multiselect">Columns:&nbsp;</label>
                     <b-form-select
+                        data-cy="column-multiselect"
```

Comment:
Do you mean for the value of the `data-cy` attribute?

Reply:
yeah, exactly. because we can chose any name here we can use one that is descriptive and makes for easy to read tests. if we end up liking that name, we could consider updating the id name eventually as well.